utils/test/check_submodule_commits

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.5
```

Comment:
Any reason to pin this to a specific python release?

Reply:
Comes from the original unsquashed code. I've removed that now.